Core/GDCore/Project/ExtensionPropertiesManager.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-2016 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#ifndef GDCORE_EXTENSIONPROPERTIESMANAGERS_H
+#define GDCORE_EXTENSIONPROPERTIESMANAGERS_H
+#include <map>
+#include "GDCore/Serialization/SerializerElement.h"
+#include "GDCore/String.h"
+
+namespace gd{
```

Comment:
Can you run VS Code "Format Document" on all C++ files that you touched in this PR?
It's powered by clang-format (you may have to install an extension), with the style defined in `.clang_format`.